OnboardIos/OnboardIos/App/AppDelegate.m

```diff
@@ -21,9 +21,21 @@ @implementation AppDelegate
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
     // Override point for customization after application launch.
     self.window = [[UIWindow alloc] initWithFrame:[[UIScreen mainScreen] bounds]];
-    [self.window setRootViewController:[ViewController new]];
-    [self.window makeKeyAndVisible];
+    //[self.window setRootViewController:[ViewController new]];
+    //[self.window makeKeyAndVisible];
     NSLog(@"%@", [Users list:@1]);
+    
+    ViewController *viewController = [[ViewController alloc] init];
```

Comment:
to simplify the size of your variable name, when you declare ViewController, you can use just *vc

And, in some cases, when you don't have any different init, you can simplify you expression as new.

So you can write the same line as: " ViewController *vc = [ViewController new];

What do you think?